ndunand/moodle-tool_mergeusers
# Search by custom user profile fields

Hi @ndunand ,

Would you like to merge this request please? We added support to make search by custom user profile fields:

- Settings support to allow search by custom user profile fields and to select desired fields.
- Support to make search by desired fields.
- Support to make direct search by desired field.
- If 'none' (default choice) is selected in the settings, the plugin should work without custom user profile support. 

How to test the new fonctionnality:

- Create  some custom user profile fields.
- Set values on some fields for desired users.
- Go to the mergeusers plugin settings and select one or more fields for which you wish to allow search.
- Open the mergeusers page and it should display options to make search by custom user profile fields allowed.
- With the search result, the process of merging users continue as usually.

Thank you.
